test/test_files/rdf/tinysnb.test

```diff
@@ -9,12 +9,9 @@
 -STATEMENT CREATE (a:R_r {iri: "a"})-[e:R_rt {iri: "b"}]->(b:R_r {iri: "c"}) RETURN a.iri, e.iri, b.iri;
 ---- 1
 a|b|c
--STATEMENT MATCH (a)-[e:knows|:R]->(b) RETURN *;
+-STATEMENT MATCH (a:organisation)-[e:knows|:R]->(b:organisation) RETURN *;
```

Comment:
Why change the test case here?

Reply:
Because resource iri table has `id` as `SERIAL`, and person table has `ID` as `INT64`, we now consider them as the same property with different type.